pkg/addons/applier.go

```diff
@@ -65,6 +66,18 @@ func (a *Applier) Outro(ctx context.Context) error {
 			return err
 		}
 	}
+
+	err = spinForInstallation(ctx, kcli)
+	if err != nil {
+		return err
+	}
+
+	if a.licenseFile != "" {
```

Comment:
maybe add a comment here about why we don't print a success message if there's no license file?

Reply:
Sure